governance/sponsors_partners.md

```diff
@@ -0,0 +1,110 @@
+(gov_sponsors_partners)=
+# Sponsors and institutional partners
+The Random Variables Council are the primary leadership for the project.
+No outside institution, individual or legal entity has the ability to
+own, control, usurp or influence the project other than by participating in the Project
+as Contributors and Council Members.
+However, because financial and institutional support is an important funding mechanism for the project,
+it is important to formally acknowledge it.
+
+ArviZ uses two figures to acknowledge financial and institutional support:
+sponsors and institutional partners. Both can have "active" and "inactive" status.
+
+We use the sponsor figure for entities who provide direct financial support,
+institutional partners for entities who employ an ArviZ contributor and
+let them work on the project as part of their official duties.
+
+(gov_sponsors)=
+## Sponsors
+Sponsors are entities that provide direct financial support.
+The minimum level of support given to ArviZ to be considered a Sponsor are:
+
+* $10,000/yr for unrestricted financial contributions (e.g., donations)
```

Comment:
Not opposed to these numbers, just curious where they came from

Reply:
Also has a point of internationalization. is $ USD?

Reply:
I copied numpy's doc and reduced the numbers to a third of the value they used, it seemed too large otherwise, even excluding czi (I hope I didn't mess my mental calculus).

Here is the reference: https://numpy.org/neps/nep-0046-sponsorship-guidelines.html

Yep, we manage the money via numfocus which is a usa entity and uses USD as main currency, I think its what makes more sense to use. I will update the sign